src/pam_hbac_ldap.c

```diff
@@ -509,6 +508,11 @@ static int secure_connection(pam_handle_t *ph,
                              const char *ca_cert,
                              bool secure)
 {
+#if defined(HPUX)
+#undef HAVE_LDAP_START_TLS                      # couldn't get it to work on hpux
```

Comment:
On the IPA side it gets an unbind request from the client:
![pcap](https://cloud.githubusercontent.com/assets/19611866/18951362/b59af6fe-863c-11e6-8d62-63d914bf1024.png)